oci-unit-tests/bind9_test.sh

```diff
@@ -0,0 +1,89 @@
+# shellcheck shell=dash
+
+# shellcheck disable=SC1090
+. "$(dirname "$0")/helper/test_helper.sh"
+. "$(dirname "$0")/helper/common_vars.sh"
+
+# cheat sheet:
+#  assertTrue $?
+#  assertEquals ["explanation"] 1 2
+#  oneTimeSetUp()
+#  oneTimeTearDown()
+#  setUp() - run before each test
+#  tearDown() - run after each test
+
+oneTimeSetUp() {
+    # Remove image before test.
+    remove_current_image
+
+    # Make sure we're using the latest OCI image.
+    docker pull --quiet "${DOCKER_IMAGE}" > /dev/null 2>&1
+
+    # Cleanup stale resources
+    tearDown
+    oneTimeTearDown
+
+    # Setup network
+    docker network create "${DOCKER_NETWORK}" > /dev/null 2>&1
+}
+
+oneTimeTearDown() {
+    docker network rm "${DOCKER_NETWORK}" > /dev/null 2>&1
+}
+
+tearDown() {
+    local container
+
+    for container in $(docker ps --filter "name=${DOCKER_PREFIX}" --format "{{.Names}}"); do
+	debug "Removing container ${container}"
+	stop_container_sync "${container}"
+    done
+}
+
+test_local_connection() {
+    local container
+
+    debug "Creating all-defaults bind9 container"
+    container=$(run_container_server)
```

Comment:
After starting the container, it is a good practice to wait for it to be ready before executing the commands.  Take a look at the `wait_.*_container_ready` family of functions for some examples.

Reply:
Thanks, fixed.